sphinx_js/doclets.py

```diff
@@ -140,6 +143,39 @@ def analyzer_for(language):
         raise SphinxError('Unsupported value of js_language in config: %s' % language)
 
 
+def process_see(doclets):
+    """Process @see tags for all doclets."""
+    for doclet in doclets:
+        if 'see' in doclet:
+            see = doclet['see']
+            process_see_list(see)
+
+
+def process_see_list(see):
+    """Process a list of @see tag values.
+
+    Leaves any value with a space as freeform text.
+    Translates other values into RST crossreferences.
+    """
+    for i, item in enumerate(see):
+        if ' ' in item:
+            item = process_see_item_text(item)
+        else:
+            item = ':any:`%s`' % item.strip()
+        see[i] = item
+
+
+def process_see_item_text(item):
```

Comment:
All the details of your processing look fine. But so far, we've pushed off all the formatting concerns like these to renderers.py and kept doclets.py about unmodified, raw doclets. I'd like to keep that distinction. Do you think you could move this machinery into the renderers module?

Naively, that'll cause some repetition, like of `see_also=process_see(doclet.get('see', []))`, across classes. This might be a good time to factor out some of the other common bits as well, like `examples=doclet.get('examples', '')` and `deprecated=doclet.get('deprecated', False),`. As a bonus, stating all that just once in the code will mean we no longer need to have a Cartesian product of tests, as you suffered from in having to test your `see` functionality 3 times.